src/LessMsi.Gui/MainForm.cs

```diff
@@ -626,8 +637,11 @@ private void InitializeComponent()
             this.btnExtractStreamFiles.Name = "btnExtractStreamFiles";
             this.btnExtractStreamFiles.Size = new System.Drawing.Size(129, 23);
             this.btnExtractStreamFiles.TabIndex = 0;
-            this.btnExtractStreamFiles.Text = "Extract Stream Files...";
+            this.btnExtractStreamFiles.Text = $"{Strings.ExtractStreamFiles}...";

```

Comment:
I think you should put the `...` in the translation itself here rather than combining translated text and a hardcoded string here. That may not be appropriate in all languages.